packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -589,11 +621,13 @@ contract OPContractsManager_Upgrade_Harness is CommonTest {
 
 contract OPContractsManager_Upgrade_Test is OPContractsManager_Upgrade_Harness {
     function test_upgradeOPChainOnly_succeeds() public {
+        if (isForkTest() && keccak256(bytes(opChain)) != keccak256(bytes("op"))) vm.skip(true);
```

Comment:
This logic repeats enough times that we should move it into a helper function in the [same location](https://github.com/ethereum-optimism/optimism/blob/8d0dd96e494b2ba154587877351e87788336a4ec/packages/contracts-bedrock/test/setup/Setup.sol#L135) as `isForkTest()` along with a comment explaining why we need it.